unoji/Model/WashTimeModel.swift

```diff
@@ -6,36 +6,52 @@
 //
 
 import Foundation
+import RealmSwift
 
-class WashTimeModel {
-    var washTime: Date? // 流すアクションをした時刻を記録
-    // 前回流すアクションをしてから現在までの経過時間
-    var elapsedTime: Int {
-        guard let washTime = washTime else {
-            return 0 }
-        return Int(abs(washTime.timeIntervalSinceNow))
+class WashTimeModel: Object {
+    @objc dynamic var washTime: Date? // 流すアクションをした時刻を記録
+    @objc dynamic var washDay: String = ""// FSCalender用に年月日を文字列に
+    @objc dynamic var elapsedTime: Int = 0// 経過時間
```

Comment:
宣言時にデフォルト値を指定した変数は型推論が効くので、明示的な型宣言はつけないことが多いです。

```suggestion
    @objc dynamic var washDay = ""// FSCalender用に年月日を文字列に
    @objc dynamic var elapsedTime = 0// 経過時間
    @objc dynamic var displayIllustration = "baby"// 経過時間に応じたイラスト名を文字列で
```